platform-hub-api/db/structure.sql

```diff
@@ -243,7 +243,7 @@ CREATE TABLE platform_themes (
     slug character varying NOT NULL,
     description text NOT NULL,
     image_url character varying NOT NULL,
-    colour character varying,
+    colour character varying NOT NULL,
```

Comment:
Not sure either. I guess migrating db generates those - maybe minor version changes behave in different way.